entities/entities/spawned_money/init.lua

```diff
@@ -30,7 +30,7 @@ end
 
 function ENT:Touch(ent)
 	-- the .USED var is also used in other mods for the same purpose
-	if ent:GetClass() ~= "spawned_money" or self.USED or ent.USED or self.hasMerged or ent.hasMerged then return end
+	if not ent.IsSpawnedMoney or self.USED or ent.USED or self.hasMerged or ent.hasMerged then return end
```

Comment:
We might want to enforce that only two spawned_money can merge. That makes most sense and doesn't have the risk of errors with other money kinds that use different variables.